frontend/src/layout/panel-layout/ProjectTree/projectTreeLogic.tsx

```diff
@@ -112,6 +114,32 @@ export const projectTreeLogic = kea<projectTreeLogicType>([
                 },
             },
         ],
+        archivedFolder: [
```

Comment:
In kea you have listeners and loaders. Loaders are usually there to load and save data like API responses. This `unfiledItems` above is a bit of a hack, as it doesn't save any data, but just returns `true`. I did it so that I could use the `unfiledItemsLoading` value that's automatically created in the `loadingPaths` selector below. However if we're doing nothing with any of the stuff loaders add, this could just be a listener.

That said, the `loadingPaths` system below is already quite legacy. We now store the loading state in `folders`... So this might need a bigger cleanup...